conf.d/ucmdb_file.yaml.example

```diff
@@ -5,6 +5,9 @@ init_config:
 instances:
   # File location where tql query result is synced from
   - location: "/path/to/file"
+    # topology synchronization interval in seconds, default is 0, meaning sync is done on each check run
+    # file_polling_interval: 0
+
```

Comment:
you added component_type_field and relation_type field, they should also be in this example config